pkg/client/rpc_client.go

```diff
@@ -263,6 +265,19 @@ func (r *RPCClient) logResult(
 			append(results, "err", err)...,
 		)
 	}
+
+	metrics.RPCCallLatency.
+		WithLabelValues(
+			"evm",                          // chain family
```

Comment:
Shouldn't we use some sort of enums/consts for chain families (ideally a single source of truth) across the entire codebase?